aissemble-open-inference-protocol-fastapi/src/aissemble_open_inference_protocol_fastapi/rest/endpoints.py

```diff
@@ -363,3 +441,28 @@ def build_inference_response(
         id=request.id,
         outputs=outputs,
     )
+
+
+def get_user_ip_from_request(request: Request):
```

Comment:
A: might need to check that this is conventional for top-level functions, but if it is recommend marking this as "internal"
```suggestion
def _get_user_ip_from_request(request: Request):
```

Reply:
Agree, this can be internal

- [x] Use python syntax for scope